<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalkulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="pasek1">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;podaj kolor&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czarny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brązowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czerwony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pomarańcz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zółty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zielony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niebieski</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fioletowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Szary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biały</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="pasek4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;podaj kolor&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czarny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brązowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czerwony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pomarańcz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zółty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zielony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niebieski</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fioletowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Szary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biały</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Złoty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sebrny</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Wartość rezystora:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="pasek3">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;podaj kolor&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czarny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brązowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czerwony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pomarańcz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zółty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zielony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niebieski</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fioletowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Szary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biały</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="pasek2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;podaj kolor&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czarny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brązowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czerwony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pomarańcz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zółty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zielony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niebieski</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fioletowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Szary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biały</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="pasek5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;podaj kolor&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brązowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czerwony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zielony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niebieski</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fioletowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Szary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Złoty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Srebrny</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Oblicz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
